tests/src/JIT/SIMD/Vector3Interop.cs

```diff
@@ -71,19 +94,38 @@ public static bool test()
             }
         }
         
+        // Argument passing test for Unix.
+        // Few arguments are passed onto stack.
+        {
+            Vector3 v1 = new Vector3(1, 2, 3);
+            Vector3 v2 = new Vector3(10, 20, 30);
+            float f0 = 0, f1 = 1, f2 = 2, f3 = 3, f4 = 4, f5 = 5, f6 = 6, f7 = 7, f8 = 8;
+            
+            float sum = nativeCall_PInvoke_Vector3Arg_Unix(
+                v1, // register
+                f0, f1, f2, f3, f4, f5, // register
+                f6, v2,  // stack
+                f7, f8); // stack
+            if (sum != 102) {
+                Console.Write("PInvoke Vector3Arg_Unix test failed\n");
+                return false;            
+            }
+        }
+        
         // JIT crashes with this testcase.
         // Disabled temporarily.
-        // {
-        //     // Expected return value = 1 + 2 + 3 = 6
-        //     Vector3 ret = nativeCall_PInvoke_Vector3Ret();
-        //     float sum = ret.X + ret.Y + ret.Z;
-        //     if (sum != 6) {
-        //         Console.WriteLine("PInvoke Vector3Ret test failed");
-        //         return false;            
-        //     }
-        // }
+        {
+            // Expected return value = 1 + 2 + 3 = 6
+            Vector3 ret = nativeCall_PInvoke_Vector3Ret();
```

Comment:
Please also compute dotProduct(ret, ret) to test that jitted code does the right thing though PInvoke is returning only 12-bytes of valid values.